python/dgl/distributed/graph_partition_book.py

```diff
@@ -830,6 +850,43 @@ def num_partitions(self):
         """
         return self._num_partitions
 
+    def init_canonical_etypes(self, etypes):
+        ''' initialize canonical etypes and related mapping.
+        '''
+        for etype, etype_id in etypes.items():
+            assert len(etype) == 3, \
+                'Canonical etype should be in format of (str, str, str).'
+            c_etype = etype
+            etype = etype[1]
+            self._etypes[etype_id] = etype
+            self._canonical_etypes[etype_id] = c_etype
+            if etype in self._etype2canonical:
+                # If one etype maps to multiple canonical etypes, empty
+                # tuple is used to indicate such ambiguity casued by etype.
+                # See more details in self.to_canonical_etype().
+                self._etype2canonical[etype] = tuple()
+            else:
+                self._etype2canonical[etype] = c_etype
+
+    @property
+    def init_by_canonical_etypes(self):
```

Comment:
For property/method that returns/is bool, use name `is_xxx` or `has_xxx`. The name here suggests it's an action.